packages/runtime/src/internal/fulfilStoryboard.ts

```
@@ -15,8 +26,196 @@
+
+function replaceSegues(
+  handler: BuiltinBrickEventHandler,
+  handlers: BrickEventsMap | BrickEventHandler[],
+  key: string | number,
+  routeParent: RouteConfOfBricks
+) {
+  let segueConf: SegueConf | undefined;
+  let segueTarget: string | undefined;
+  if (
+    Array.isArray(handler.args) &&
+    ((segueTarget = handler.args[0] as string),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)